detection-rules/attachment_html_smuggling_atob.yml

```diff
@@ -1,22 +1,29 @@
 name: "Attachment: HTML smuggling with atob and high entropy"
 description: |
   Recursively scans files and archives to detect HTML smuggling techniques.
+references:
+  - "https://delivr.to/payloads?id=7dbf0d83-1557-4345-bf67-d18c4256b0c1"
 type: "rule"
 severity: "high"
 source: |
   type.inbound
-  and any(attachments, .size <= 60000 and
+  and any(attachments, .size <= 20000 and
       (
           .file_extension in~ ("html", "htm", "shtml", "dhtml") or
           .file_extension in~ $file_extensions_common_archives or
           .file_type == "html"
       )
       and any(beta.binexplode(.),
-          .scan.entropy.entropy >= 5
+          .scan.entropy.entropy >= 4.5
           and (
-              all(["document", "write", "atob"], . in ..scan.javascript.identifiers)
-              
+              all(["download", "document", "write", "atob"], . in ..scan.javascript.identifiers)
               // usage: document['write'](atob)
+              or any(.scan.strings.strings, strings.ilike(., "*document*write*atob*"))
+              // usage: some_var = atob();
+              or any(.scan.strings.strings, strings.ilike(., "*=*atob*;"))
+              or any(.scan.strings.strings, strings.ilike(., "*a.download*=*"))
```

Comment:
```suggestion
```
pulling download as it doesn't match rule intent (i know it was my original suggestion) :)